core/src/styles/core/_nav.scss

```diff
@@ -4,13 +4,12 @@
 nav {
   hr,
   .cat-hr {
-    margin-top: $cat-nav-padding-horizontal * 0.5;
-    margin-bottom: $cat-nav-padding-horizontal * 0.5;
+    margin: cat-token('size.spacing.xs') calc(-1 * $cat-nav-padding-horizontal);
```

Comment:
spacing values are deprecated as tokens. they only exist to generate the correct spacing classes. Use hard-coded spacings here.